src/components/submission/decrypt.vue

```diff
@@ -350,9 +350,9 @@ $actions-padding-left: $label-icon-max-width + $margin-right-icon;
     "field": {
       "splitSelectMultiples": "Split “select multiple” choices into columns",
       "removeGroupNames": "Remove group names",
-      "deletedFields": "Include fields not in the published version of the Form"
+      "deletedFields": "Include fields not in the published Form"
     },
-    "deletedFieldsHelp": "Use this option if you need to see data collected with previous versions of this Form.",
+    "deletedFieldsHelp": "Use this option if you need to see fields referenced in previous Form versions.",
```

Comment:
What do you think of referenced -> only? This is a bit challenging to express precisely!

Reply:
I agree that it's challenging to express this precisely! Here I'm using the text that @yanokwa suggested in Slack (on June 7). I don't have strong feelings myself, but I personally think that either way would be reasonably clear.

Reply:
Let's leave it. With "only" there are two readings: "just the fields in older versions, not the ones in the current version" or "fields that only existed in previous versions and now don't exist"

I don't really know what "referenced" means but I think you're right it's clear enough.